peps/pep-0781.rst

```diff
@@ -0,0 +1,87 @@
+PEP: 781
+Title: Adding __type_checking__ constant
+Author: Inada Naoki <songofacandy@gmail.com>
+Discussions-To: https://discuss.python.org/t/85728
+Status: Draft
+Type: Standards Track
+Topic: Typing
+Created: 24-Mar-2025
+Python-Version: 3.14
+
+
+Abstract
+========
+
+This PEP proposes adding a new keyword ``__type_checking__`` that serves the
+same purpose as ``typing.TYPE_CHECKING``.
```

Comment:
```suggestion
same purpose as :data:`typing.TYPE_CHECKING`.
```

Reply:
Also I think it would be great to link to the keyword definition:

```rst
This PEP proposes adding a new ``__type_checking__`` :ref:`keyword <python:keywords>` that serves the
```